package.json

```diff
@@ -156,6 +156,10 @@
         "yarn format",
         "git add"
       ],
+      "*.json": [
+        "node utils/check-mismatched-dependencies.js",
```

Comment:
We should be running check-lockfile at this step too. It's important because otherwise we end up with artifactory URLs in github

Reply:
That looks like it is handled in the `yarn.lock` step. Should we also add that here?